skia/BUILD.gn

```diff
@@ -0,0 +1,39 @@
+import("//react-native/common.gni")
+config("LibRnRNKeyEvent_config") {
+  cflags_cc = [ "-std=c++17","-g"]
+  cflags_cc += [
+    "-Wno-extra-semi",
+    "-Wno-sign-compare",
+    "-Wno-header-hygiene",
+  ]
+
+  include_dirs = [
+    "./",
+    "//react-native/ReactCommon/",
+  ]
+}
+
+rn_cxx_component("RnRNKeyEvent") {
```

Comment:
Is RNKeyEvent or RnKeyEvent is not enough?, ANy specific reason to have to  **RnRN**KeyEvent